From 81cd657b104ed808ebb1a763b2755411a26249d2 Mon Sep 17 00:00:00 2001
From: stbenz <benz.st@gmail.com>
Date: Fri, 26 Jun 2015 22:10:55 +0200
Subject: [PATCH] remove -Werror

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 07d6965..1e69ae4 100644
--- a/Makefile
+++ b/Makefile
@@ -28,7 +28,7 @@ PROG    := $(BUILDDIR)/tvheadend
 #
 
 CFLAGS  += -g -O2 -Wunused-result
-CFLAGS  += -Wall -Werror -Wwrite-strings -Wno-deprecated-declarations
+CFLAGS  += -Wall -Wwrite-strings -Wno-deprecated-declarations
 CFLAGS  += -Wmissing-prototypes
 CFLAGS  += -fms-extensions -funsigned-char -fno-strict-aliasing
 CFLAGS  += -D_FILE_OFFSET_BITS=64
-- 
1.9.1

