diff -Naur libtheora-1.1.1-old/configure.ac libtheora-1.1.1/configure.ac
--- libtheora-1.1.1-old/configure.ac	2014-09-18 21:12:56.481476927 +0200
+++ libtheora-1.1.1/configure.ac	2014-09-18 21:14:39.025481179 +0200
@@ -11,6 +11,7 @@
 
 AM_CONFIG_HEADER([config.h])
 AC_CONFIG_SRCDIR([lib/fdct.c])
+AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE
 AM_MAINTAINER_MODE
 
@@ -61,7 +62,6 @@
 AM_PROG_LIBTOOL
 
 dnl Add parameters for aclocal
-AC_SUBST(ACLOCAL_AMFLAGS, "-I m4")
 
 dnl Check for doxygen
 AC_CHECK_PROG(HAVE_DOXYGEN, doxygen, true, false)
diff -Naur libtheora-1.1.1-old/Makefile.am libtheora-1.1.1/Makefile.am
--- libtheora-1.1.1-old/Makefile.am	2014-09-18 21:12:56.477476927 +0200
+++ libtheora-1.1.1/Makefile.am	2014-09-18 21:15:10.713482492 +0200
@@ -8,6 +8,8 @@
 EXAMPLES_DIR =
 endif
 
+ACLOCAL_AMFLAGS = -I m4
+
 SUBDIRS = lib include doc tests m4 $(EXAMPLES_DIR)
 
 
